cicd/gitops/monitoring-argo-cd-custom-resource-workloads.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: ASSEMBLY
+[id="monitoring-argo-cd-custom-resource-workloads"]
+= Monitoring Argo CD custom resource workloads
+include::_attributes/common-attributes.adoc[]
+:context: monitoring-argo-cd-custom-resource-workloads
+
+toc::[]
+
+[role="_abstract"]
+With {gitops-title}, you can monitor the availability of Argo CD custom resource workloads for specific Argo CD instances. By monitoring Argo CD custom resource workloads, you have the latest information about the state of your Argo CD instances by enabling alerts for them. The Operator then triggers the alerts when the component workload pods such as application-controller, repo-server, or server of the correpsonding Argo CD instance are unable to come up for certain reasons and there is a drift between the number of ready replicas and the number of desired replicas for a certain period of time.
```

Comment:
The last sentence here is hard to parse when reading it. Maybe simple re-organizing it may help:

 When the component workload pods such as application-controller, repo-server, or server of the correpsonding Argo CD instance are unable to come up for certain reasons and there is a drift between the number of ready replicas and the number of desired replicas for a certain period of time, the Operator then triggers the alerts.
 
 Or split it up into multiple sentences...

Reply:
Addressed it.